src/components/simulator/__snapshots__/LogEvent.test.tsx.snap

```diff
@@ -0,0 +1,324 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
big fan of snapshot testing now - it's the only sensible way to check HTML outputs

Reply:
Totally agree. Also, while probably not a huge deal for the log event stuff, but as a general rule for snapshot testing, I try to have pertinent asserts along with the snapshot. Otherwise, it can be really easy to update a snapshot without realizing you are changing an expected truth.

Here is an example:
https://github.com/nyaruka/floweditor/pull/801/files#diff-05f981483ae9b76ba83005bfafd192b3R45

Basically, just treat the snapshot as a bonus assertion.